class Failure {
  final String message;

  Failure(
      [this.message = "An unexpected error occurred. Please try again later."]);
}
